Smart-Table.debug.js

```diff
@@ -456,7 +467,8 @@
 
                 var dataRow, oldValue;
 
-                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' && selectionMode !== 'single')) {
+                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' 
+                    && selectionMode !== 'single' && selectionMode !== 'customMultiple')) {
```

Comment:
I didn't change this to use isModeMultiple() because the function takes a parameter "selectionMode" instead of using the scope's selectionMode.